### PR TITLE
Add info for linking JS to pages as well as a few tips!

### PR DESCRIPTION
**Pull request by CatQueenCodes**
We never mention how to link js files. 
In the google homepage project (https://www.theodinproject.com/courses/foundations/lessons/html-css) we have a **Try These Before Starting** section that includes "Identify three ways you can include your CSS styles in a page", so I added similar to RPS (1st JS project). A lot of students don't know to use externally linked js files. I have noticed a lot don't know how to test their code either as they build or use the console so I added a few tips on that.

### CODE REVIEW

File: foundations/javascript_basics/project_rock_paper_scissors.md [hunk 1]
```diff
@@ -2,15 +2,20 @@
 
 We're going to make a simple implementation of grade-school classic "rock paper scissors". If you don't know what that is check the [Wikipedia article](https://en.wikipedia.org/wiki/Rock%E2%80%93paper%E2%80%93scissors) or [this](https://www.wikihow.com/Play-Rock,-Paper,-Scissors) ridiculous step-by-step. For the moment we're just going to play the game from the browser console, but we will revisit it and add a front end later so don't forget to keep the code on GitHub! You might notice some 'view in browser' links in the student solutions - this is coming in a later lesson. When you get there don't forget to come back and add your link!
 
-Finally, this is your first JavaScript program built from scratch, so don't forget the previous lesson on problem solving and plan your solution before writing any code.
+### Try These Before Starting
```
Comment (leila-alderman) [inline foundations/javascript_basics/project_rock_paper_scissors.md:5]:
```suggestion
### Quick Exercises Before Starting
```

File: foundations/javascript_basics/project_rock_paper_scissors.md [hunk 2]
```diff
@@ -2,15 +2,20 @@
...
+
+1.  Identify three ways you can include your JavaScript in a page
```
Comment (leila-alderman) [inline foundations/javascript_basics/project_rock_paper_scissors.md:7]:
```suggestion
1. Identify three ways to include JavaScript in a page.
```

File: foundations/javascript_basics/project_rock_paper_scissors.md [hunk 3]
```diff
@@ -2,15 +2,20 @@
...
+2. Test it out! Write `console.log("Hello World");` in JavaScript and check to see if it displays in the browsers console.
```
Comment (leila-alderman) [inline foundations/javascript_basics/project_rock_paper_scissors.md:8]:
```suggestion
2. Test it out! Write `console.log("Hello World");` in JavaScript and check to see if it displays in the browser's console.
```

File: foundations/javascript_basics/project_rock_paper_scissors.md [hunk 4]
```diff
@@ -2,15 +2,20 @@
...
+3. If you haven't used it already, this is where the [VSCode Live Server extension](https://www.theodinproject.com/courses/foundations/lessons/text-editors#additional-set-up-live-server-extension-for-vscod) becomes handy! 
+
+Finally, this is your first JavaScript program built from scratch, so don't forget the previous lesson on problem solving and plan your solution before writing any code. Test each piece as you build to ensure it is working before moving to the next!
```
Comment (leila-alderman) [inline foundations/javascript_basics/project_rock_paper_scissors.md:11]:
```suggestion
Finally, this is your first JavaScript program built from scratch, so don't forget the previous lesson on problem solving. Plan your solution out before writing any code, and test each piece as you build to ensure it is working before moving on to the next!
```

Yes, I do realize that you didn't change this at all... itchy fingers...

**Review by leila-alderman — COMMENTED** [review]
A few suggestions for wording, but great addition!

File: foundations/javascript_basics/project_rock_paper_scissors.md (hunk 4 above)
Comment (CatQueenCodes) [reply foundations/javascript_basics/project_rock_paper_scissors.md:11]: No problem! Better to get it fixed now since its right next to my edit.

**Review by leila-alderman — APPROVED** [review]

**Pull request merged**